src/plugins/hetero/src/plugin.cpp

```diff
@@ -96,6 +96,15 @@ std::pair<ov::SupportedOpsMap, ov::hetero::SubgraphsMappingInfo> ov::hetero::Plu
     ov::SupportedOpsMap supported_ops_final;
     std::map<std::string, ov::SupportedOpsMap> query_results;
     ov::hetero::SubgraphsMappingInfo mapping_info;
+    ResultVector new_outputs;
+    for (auto& param : model->get_parameters()) {
+        if (param->get_users().size() == 0) {
+            auto result = std::make_shared<ov::op::v0::Result>(param);
+            ov::copy_runtime_info(param->shared_from_this(), result);
+            new_outputs.push_back(result);
+        }
+    }
+    model->add_results(new_outputs);
     for (const auto& device_name : device_names) {
```

Comment:
is this happening because of model splitting? wonder why we would include this *independent* param to subgraph if no relationship with other ops in the subgraph?

Reply:
If a original model with a independent param, Hetero will package it as a subgraph, but it has no result ops, so should copy itself as a result ops.